<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>740</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>955</width>
    <height>740</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>955</width>
    <height>740</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Face id</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
background: rgb(200, 0, 45);
opacity: 3;
border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="camera">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>530</y>
         <width>91</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
    background-color: white;
    border-radius: 30px;

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="opencv_label">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>641</width>
         <height>471</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
    border: 3px solid #fff; /* Белая рамка */</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="filters">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>540</y>
         <width>71</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
    background-color: white;
    border-radius: 30px;
    margin: 4px 2px;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="bd_of_people">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>530</y>
         <width>101</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background: white;
    border-radius: 30px;
    margin: 4px 2px;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>85</width>
         <height>85</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
